### PR TITLE
Updated classes, added tests for addEto command

### LINKED ISSUE #116: Add and update tests for classes related to add Event command

### PR DESCRIPTION
**Pull request by Eriksen2411**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddEventCommandParser.java [hunk 1]
```diff
@@ -32,25 +32,19 @@ public AddEventCommand parse(String args) throws ParseException {
...
+                || argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddEventCommand.MESSAGE_USAGE));
         }
 
-        Index index;
-
-        try {
-            index = ParserUtil.parseIndex(argMultimap.getPreamble());
-        } catch (ParseException e) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddEventCommand.MESSAGE_USAGE), e);
```
Comment (vevek) [inline src/main/java/seedu/address/logic/parser/AddEventCommandParser.java:44]: I think it is better to have it in the previous form, so that the ParseException e's message can be shown to the user as well.

Comment (vevek) [reply src/main/java/seedu/address/logic/parser/AddEventCommandParser.java:44]: Although the new design does simplify things.

File: src/test/java/seedu/address/logic/commands/AddEventCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        AddEventCommand addEventCommand = new AddEventCommand(INDEX_THIRD, eventToAdd);
+
+        assertThrows(
+                CommandException.class,
+                Messages.MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX, () -> addEventCommand.execute(model)
+        );
+    }
+
+    @Test
+    public void execute_duplicatePerson_throwsCommandException() {
```
Comment (vevek) [inline src/test/java/seedu/address/logic/commands/AddEventCommandTest.java]: The method header name here is wrong. For other methods, your method header names also seem irrelevant to the what the test actually does.

File: src/test/java/seedu/address/logic/commands/AddEventCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import seedu.address.model.task.Interval;
+import seedu.address.model.task.repeatable.Event;
+import seedu.address.testutil.EventBuilder;
+
+public class AddEventCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), getTypicalProjectsFolder(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() throws Exception {
```
Comment (vevek) [inline src/test/java/seedu/address/logic/commands/AddEventCommandTest.java]: Wrong method header.

File: src/test/java/seedu/address/logic/commands/AddEventCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        Event validEvent = new EventBuilder().withDescription("CS2106 Tutorial")
+                .withAtDate(LocalDate.of(2020, 01, 01)).withInterval(Interval.WEEKLY).build();
+
+        CommandResult commandResult = new AddEventCommand(INDEX_FIRST, validEvent).execute(model);
+
+        assertEquals(String.format(Messages.MESSAGE_ADD_EVENT_SUCCESS, validEvent), commandResult.getFeedbackToUser());
+    }
+
+    @Test
+    public void execute_invalidProjectIndex_throwsCommandException() {
```
Comment (vevek) [inline src/test/java/seedu/address/logic/commands/AddEventCommandTest.java:41]: Irrelevant method header.

File: src/test/java/seedu/address/logic/parser/AddEventCommandParserTest.java [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.AddEventCommand;
+import seedu.address.model.task.Interval;
+import seedu.address.model.task.repeatable.Event;
+import seedu.address.testutil.EventBuilder;
+
+public class AddEventCommandParserTest {
```
Comment (vevek) [inline src/test/java/seedu/address/logic/parser/AddEventCommandParserTest.java:27]: Header names here are fine

**Review by vevek — CHANGES_REQUESTED** [review]
Some headers need to be double checked

File: src/main/java/seedu/address/logic/parser/AddEventCommandParser.java (hunk 1 above)
Comment (Eriksen2411) [reply src/main/java/seedu/address/logic/parser/AddEventCommandParser.java:44]: ah for this actually the invalid format is thrown above
the parseIndex has its own exception throw for invalid Index. So no problem with this, and actually I change it because if invalid index, the correct exception should be thrown (invalid index), not (invalid fomat)

File: src/test/java/seedu/address/logic/commands/AddEventCommandTest.java (hunk 3 above)
Comment (Eriksen2411) [reply src/test/java/seedu/address/logic/commands/AddEventCommandTest.java]: resolved

File: src/main/java/seedu/address/logic/parser/AddEventCommandParser.java (hunk 1 above)
Comment (vevek) [reply src/main/java/seedu/address/logic/parser/AddEventCommandParser.java:44]: Oh okay. In that case, nice catch :)

File: src/test/java/seedu/address/logic/commands/AddEventCommandTest.java (hunk 2 above)
Comment (Eriksen2411) [reply src/test/java/seedu/address/logic/commands/AddEventCommandTest.java]: resolved

**Review by vevek — APPROVED** [review]
LGTM
